typed-builder-macro/src/field_info.rs

```diff
@@ -73,6 +76,26 @@ impl<'a> FieldInfo<'a> {
         }
     }
 
+    pub fn setter_method_name(&self) -> Ident {
+        let (name, was_stripped) = strip_raw_ident_prefix_with_status(self.name.to_string());
```

Comment:
You don't actually need `was_stripped`. You only need to re-add the `r#` if both the prefix and the suffix are `None`, but in that case you can just return `self.name` (`.clone()`)